akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -0,0 +1,89 @@
+# Routers
+
+## Dependency
+
+To use Akka Actor Typed, you must add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-actor-typed_$scala.binary_version$
+  version=$akka.version$
+}
+
+## Introduction
+
+In some cases it is useful to distribute messages of the same type over a set of actors, so that messages can be 
+processed in parallel - a single actor will only process one message at a time. 
```

Comment:
Can we call back to untyped routers here so that it is clearer this is generally the same use case for actor routers?

Reply:
Perhaps sprinkle in the untyped router doc like `@@include[my-file.md](includes/my-file.md)`, where applicable?

Reply:
I don't agree we want to direct users from here to there. If you know untyped routers you already know, and if you don't I think it confuses more than helps reading all those docs given that those routers have different APIs and work differently.

Reply:
I agree, let's think about linking between the two when we do restructuring work. I created ticket https://github.com/akka/akka/issues/26487

Reply:
I mean that I agree with Johan, but let's do the holistic thinking later.

Reply:
Oh that's true, good point.